otb-luci/files/usr/lib/lua/luci/view/otb_overview.htm

```diff
@@ -72,6 +94,219 @@ <h2><%:Overview%></h2>
         return ret + seconds + ' seconds';
     }
 
+    /* start of otb-get-data section */
+    XHR.poll(15, '<%=url("admin/overthebox/data")%>', null, function(x,p) {
+        _otb_data = p;
+    });
+    function otb_get_data(...args) {
+        while(_otb_data.time == 0);
+        var data = _otb_data;
+        for(var i=0; i < args.length; i++)
+            if (data && (args[i] in data))
+                data = data[args[i]];
+            else
+                return false;
+        return data;
+    }
+    /* end of otb-get-data section */
+
+    /* start of overview-graph config */
+    var graphs = [
+        {plot: 'rx'}, {plot: 'tx'}
+    ];
+
+    interface_colors = (str) => {
+        if(str == 'if1')
+            return 'rgb(57, 106, 177)';
+        if(str == 'if2')
+            return 'rgb(218, 124, 48)';
+        if(str == 'if3')
+            return 'rgb(62, 150, 81)';
+        if(str == 'if4')
+            return 'rgb(204, 37, 41)';
+        if(str == 'if5')
+            return 'rgb(83, 81, 84)';
+        if(str == 'if6')
+            return 'rgb(107, 76, 154)';
+        if(str == 'if7')
+            return 'rgb(146, 36, 40)';
+        if(str == 'if8')
+            return 'rgb(148, 139, 61)';
+    }
+
+    var graphs_data_map = {
+        date: {
+            type: 'timeCat',
+            formatter: function formater(val){ return ((_overview_graph_data[_overview_graph_data.length-1].date.getTime() - val)/1000) + ' s' }
+        },
+        tx: {
+            tickCount: 4,
+            formatter: function formater(val){ return prettySize(val) },
+            alias: 'Upload'
+        },
+        tx_max: {
+            tickCount: 4,
+            formatter: function formater(val){ return prettySize(val) },
+            alias: 'Upload Max'
+        },
+        rx: {
+            tickCount: 4,
+            formatter: function formater(val){ return prettySize(val) },
+            alias: 'Download'
+        },
+        rx_max: {
+            tickCount: 4,
+            formatter: function formater(val){ return prettySize(val) },
+            alias: 'Download Max'
+        },
+        rtt:{
+            alias: 'Ping',
+            formatter: function formater(val){ return val + ' ms' },
+            min: 0
+        },
+        rtt_var: {
+            alias: 'Jitter',
+            formatter: function formater(val){ return val + ' ms' },
+            min: 0
+        }
+    }
+
+    var _overview_graph_timeout = 60*1000;
```

Comment:
What's the unit ? What is it used for ? A comment might be useful

Reply:
It is used to clean records older than 60s unless you want graphs so narrowed that become unreadable